src/backend_model_instance.cc

```diff
@@ -962,5 +962,24 @@ TRITONBACKEND_ModelInstanceReportBatchStatistics(
   return nullptr;  // success
 }
 
+TRITONAPI_DECLSPEC TRITONSERVER_Error*
+TRITONBACKEND_ModelInstanceReportResponseStatistics(
+    TRITONBACKEND_ModelInstance* instance, TRITONBACKEND_Response* response,
+    const bool success, const uint64_t response_start,
+    const uint64_t compute_output_start, const uint64_t response_end,
+    const uint32_t response_flags)
+{
+#ifdef TRITON_ENABLE_STATS
+  TritonModelInstance* ti = reinterpret_cast<TritonModelInstance*>(instance);
+  InferenceResponse* ir = reinterpret_cast<InferenceResponse*>(response);
+
+  INFER_STATS_DECL_TIMESTAMP(response_end_ns);
```

Comment:
`response_end` and `response_end_ns` is confusing. Do we expect a lot of difference between the timestamps currently being captured here?
It is kind of user-dependent by the fact when exactly they call TRITONBACKEND_ModelInstanceReportResponseStatistics, but do we care to capture `response_end_ns`?

Reply:
I think compute_output_end should be treated as response_end.

Reply:
Updated.